install/kubernetes/cilium/values.yaml

```diff
@@ -412,7 +412,8 @@ enableK8sEventHandover: false
 # TODO: Add documentation
 # enableIdentityMark: false
 
-# enableK8sEndpointSlice: false
+# -- Enable CiliumEndpointSlice feature.
+enableCiliumEndpointSlice: false
```

Comment:
Why is this removing the `enableK8sEndpointSlice` comment?

Reply:
Thanks for catching, i think i got confused with `enableK8sEndpointSlice` with `enableCiliumEndpointSlice`.  I thought  we have only single EndpointSlice i.e. `enableCiliumEndpointSlice`.  Hence removed the `enableK8sEndpointSlice` and added `enableCiliumEndpointSlice`.

Anyways fixed now.